sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkLauncher.java

```diff
@@ -762,14 +817,9 @@ private String logsDir(long now) {
   static final DoFn<Event, byte[]> EVENT_TO_BYTEARRAY =
           new DoFn<Event, byte[]>() {
             @ProcessElement
-            public void processElement(ProcessContext c) {
-              try {
-                byte[] encodedEvent = CoderUtils.encodeToByteArray(Event.CODER, c.element());
-                c.output(encodedEvent);
-              } catch (CoderException e1) {
-                LOG.error("Error while sending Event {} to Kafka: serialization error",
-                        c.element().toString());
-              }
+            public void processElement(ProcessContext c) throws IOException {
```

Comment:
I don't think it will break the exists tests. These are load tests I don't think are expected to have invalid records. The problem with handling it specifically is that it will result in more of such excess exception handling in other places and user's Beam applications.